app/views/samples/_plate_create.html.erb

```diff
@@ -1,29 +1,135 @@
 <div class="container">
+  <br>
+
+  <div class="row">
+    <div class="col-md-6">
+      <h3>1) Scan barcode</h3>
+      <hr>
+      <div id="loadingMessage">🎥 Unable to access video stream (please make sure you have a webcam enabled)</div>
+      <canvas id="canvas" hidden></canvas>
+    </div>
+    <div class="col-md-6">      
+      <h3>2) Insert sample into well</h3>
+      <hr>
+      <div id="output" class="text-center" hidden>
+        <br>
+        <h3 hidden>Place sample:<pre id="outputData"></pre>into well:<pre id="wellToInsertSample"></pre></h3>
+        <br><br><br>
+        <button id="nextSampleButton" type="button" class="btn btn-success btn-block btn-lg">Next Sample</button>
+      </div>
+    </div>
+  </div>
+
+  <br><br>
+  <h3>Plate Status</h3>
+  <hr>
   <%= form_for(:samples, url: step3_sample_bulkprepared_path) do |form| %>
     <table class="table table-hover table-bordered">
-      <thead class="thead-dark text-center">
-      <tr>
-        <th></th>
-        <% (1..12).each do |col| %>
-          <th><%= col %></th>
-        <% end %>
-      </tr>
-      </thead>
-      <tbody>
-        <% ('A'..'H').each do | row| %>
-          <tr>
-            <td class="dark-cell text-center"><%= row %></td>
-            <% (1..12).each do |col| %>
-              <%= hidden_field_tag 'wells[][row]', row  %>
-              <%= hidden_field_tag 'wells[][col]', col  %>
-              <td><%= select_tag("wells[][id]", options_from_collection_for_select(@samples, "id", "id"), { include_blank: true , class: 'form-control'}) %></td>
-            <% end %>
-          </tr>
-        <% end %>
-      </tbody>
-    </table>
-    <div class="actions">
-      <%= form.submit "Create Plate", :class => 'btn btn-dark float-right' %>
-    </div>
-  <% end %>
-</div>
\ No newline at end of file
+        <thead class="thead-dark text-center">
+        <tr>
+          <th><!-- row letter --></th>
+          <% (1..12).each do |col| %>
+            <th><%= col %></th>
+          <% end %>
+        </tr>
+        </thead>
+        <tbody>
+          <% ('A'..'H').each do |row| %>
+            <tr>
+              <td class="dark-cell text-center"><%= row %></td>
+              <% (1..12).each do |col| %>
+                <td>
+                  <%= hidden_field_tag 'wells[][row]', row  %>
+                  <%= hidden_field_tag 'wells[][col]', col  %>
+                  <input type="text" class="form-control">
+                </td>
+              <% end %>
+            </tr>
+          <% end %>
+        </tbody>
+      </table>
+      <div class="actions">
+        <%= form.submit "Create Plate", :class => 'btn btn-dark float-right' %>
+      </div>
+    <% end %>
+</div>
+<br><br>
+
+<script src="https://cozmo.github.io/jsQR/jsQR.js"></script>
+<script>
```

Comment:
add this as a javascript file in the assets folder, you can include here specifically with a tag like this
`<%= javascript_include_tag 'https://cozmo.github.io/jsQR/jsQR.js', 'data-turbolinks-track': 'reload' %>`